app/code/BVMyBlog/Blog/Controller/Index/Index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Controller\Index;
+
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+
+/**
+ * Class Index
+ *
+ * Controller Index
+ */
+class Index extends Action implements HttpGetActionInterface
+{
+    protected $_pageFactory;
```

Comment:
Why is the field **protected**? Please **do not** use underscores in field, argument names